doc/nrf/known_issues.rst

```diff
@@ -2411,19 +2448,20 @@ The issues in this section are related to components provided in :ref:`nrfxlib`.
 
 Crypto
 ======
-.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
 
-NCSDK-20688: For ChaCha-Poly, incorrect tag will be produced if plaintext is empty
+The issues in this section are related to :ref:`nrfxlib:crypto`.
+
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+NCSDK-21430: For ChaCha-Poly, incorrect tag will be produced if plaintext is empty
```

Comment:
This entry is missing a qualifying statement on scope, this is only related to nrf_cc3xx

```
NCSDK-21430: For ChaCha20/Poly1305 using nrf_cc3xx, incorrect tag will be produced if plaintext is empty
For encryption, empty plaintext will result in incorrect tag.
For decryption, correct tags will not be accepted.
```